lib/bolt/config.rb

```diff
@@ -128,7 +128,10 @@ def self.load_bolt_defaults_yaml(dir)
 
       # Validate the config against the schema. This will raise a single error
       # with all validation errors.
-      Validator.new.validate(data, defaults_schema, filepath)
```

Comment:
I know it's going away soon but we should also add this to the `from_file` handler so that `--configfile` also issues this warning

Reply:
Added to https://github.com/puppetlabs/bolt/pull/2386 already.